examples/design-system/src/tokenami.css

```diff
@@ -9,13 +9,24 @@
     }
   }
 
-  :root {
+  :root, .theme-light {
     --_grid: .25rem;
-    --color_slate-100: #f1f5f9;
+    --anim_wiggle: wiggle 1s ease-in-out infinite;
+    --color_primary: #f1f5f9;
+    --color_secondary: #334155;
     --color_slate-700: #334155;
-    --radii_rounded: 10px;
     --font_sans: sans-serif;
+    --radii_rounded: 10px;
+  }
+
+  .theme-dark {
+    --_grid: .25rem;
     --anim_wiggle: wiggle 1s ease-in-out infinite;
+    --color_primary: #0ea5e9;
+    --color_secondary: #f1f5f9;
+    --color_slate-700: #334155;
+    --font_sans: sans-serif;
```

Comment:
i'd like to clean this up so that if themes share vars, the stylesheet chains selectors to keep file size small:

```css
  :root, .theme-light, .theme-dark {
    --_grid: .25rem;
    --anim_wiggle: wiggle 1s ease-in-out infinite;
    --font_sans: sans-serif;
    --radii_rounded: 10px;
  }
  
  .theme-light {
    --color_primary: #f1f5f9;
    --color_secondary: #334155;
    --color_slate-700: #334155;
  }
  
  .theme-dark {
    --color_primary: #0ea5e9;
    --color_secondary: #f1f5f9;
    --color_slate-700: #334155;
  }
```